examples/jest/__tests__/catAPI.spec.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+const path = require('path')
+const { Pact, Matchers } = require('../../../dist/pact')
+const getMeCats = require('../').getMeCats
+
+describe("Cat's API", () => {
+  let url = 'http://localhost'
+
+  describe("another works", () => {
+    it('returns a successful body', async () => {
+      // add expectations
+      const EXPECTED_BODY = [
+        { cat: 2 }, { cat: 3 }
+      ]
+
+      const interaction = {
+        state: 'i have a list of cats',
+        uponReceiving: 'a request for cats with given catId',
+        withRequest: {
+          method: 'GET',
+          path: '/cats',
+          query: {
+            'catId[]': Matchers.eachLike('1'),
+          },
+          headers: {
+            'Accept': 'application/json'
+          }
+        },
+        willRespondWith: {
+          status: 200,
+          headers: {
+            'Content-Type': 'application/json'
+          },
+          body: EXPECTED_BODY
+        }
+      }
+
+      await provider.addInteraction(interaction)
```

Comment:
Similarly to the comment above, I personally think that the `.then()` style is more readable, as it lets you separate the When from the Then.